test/hotspot/gtest/nmt/test_vmatree.cpp

```diff
@@ -179,11 +312,15 @@ TEST_VM_F(NMTVMATreeTest, OverlappingReservationsResultInTwoNodes) {
   EXPECT_EQ(2, count_nodes(tree));
 }
 
-TEST_VM_F(NMTVMATreeTest, UseFlagInplace) {
+TEST_VM_F(NMTVMATreeTest, UseTagInplace) {
   Tree tree;
   VMATree::RegionData rd1(si[0], mtTest);
   VMATree::RegionData rd2(si[1], mtNone);
```

Comment:
We keep using those later in and it is hard to keep track of what `rd1` is. I suggest we use better names for those, such as `rd_Test`, then I know what mem_tag that region uses without having to go back.

I reman other instances, like `call_stack_2` as well.

I have changed lots of those and here is a diff file with this change: [gerard.diff.zip](https://github.com/user-attachments/files/20376150/gerard.diff.zip)

Reply:
Unfortunately, your patch couldn't be applied, since it cannot find some expected text in my local code.

Reply:
It's a lot easier to apply a patch by rebasing with git branches. Send the branch that's on your github instead.

Reply:
Thanks for the advice. I have already made the changes in the patch in my own code.